### PR TITLE
Make VCard code slightly clearer and avoid bug with semicolons in names

### PR DESCRIPTION
**Pull request by expectocode**

### CODE REVIEW

File: telethon/telegram_client.py [hunk 1]
```diff
@@ -51,7 +51,6 @@
     PeerUser, InputPeerUser, InputPeerChat, InputPeerChannel)
 from .tl.types.messages import DialogsSlice
 
-
```
Comment (Lonami) [inline telethon/telegram_client.py]: Why remove this line?

File: telethon/telegram_client.py [hunk 2]
```diff
@@ -803,12 +802,18 @@ def _download_contact(mm_contact, file):
             f = file
 
         try:
+            # Remove these pesky characters
+            first_name = first_name.replace(';','')
+            if last_name is None:
```
Comment (Lonami) [inline telethon/telegram_client.py]: Mouthful. Maybe `(last_name or '').replace(';', '')` works better.

File: telethon/telegram_client.py [hunk 3]
```diff
@@ -803,12 +802,18 @@ def _download_contact(mm_contact, file):
...
+                last_name = ''
+            else:
+                last_name = last_name.replace(';','')
```
Comment (Lonami) [inline telethon/telegram_client.py]: Have you checked for other kind of characters that would be illegal, besides removing the `;`? What happens if the name is all `;` and so both strings end empty? Is that valid? Or should we remove the name field?

File: telethon/telegram_client.py (hunk 2 above)
Comment (expectocode) [reply telethon/telegram_client.py]: Yeah, can do.

File: telethon/telegram_client.py (hunk 1 above)
Comment (expectocode) [reply telethon/telegram_client.py]: For some reason my editor didn't like it ¯\_(ツ)_/¯

File: telethon/telegram_client.py (hunk 3 above)
Comment (expectocode) [reply telethon/telegram_client.py]: I found this: https://tools.ietf.org/html/rfc6350#section-3.4 which explains precisely what needs to be escaped, and there's a lot more to fix. Perhaps using a VCard library would be easier?

Comment (Lonami) [reply telethon/telegram_client.py]: Uh, maybe. How many people wants to save contacts like this anyway?

File: telethon/telegram_client.py (hunk 1 above)
Comment (Lonami) [reply telethon/telegram_client.py]: You need to escape the shrug.

File: telethon/telegram_client.py [hunk 4]
```diff
@@ -803,12 +802,18 @@ def _download_contact(mm_contact, file):
...
             f.write('BEGIN:VCARD\n')
             f.write('VERSION:4.0\n')
             f.write('N:{};{};;;\n'.format(
-                first_name, last_name if last_name else '')
+                first_name, last_name)
             )
-            f.write('FN:{}\n'.format(' '.join((first_name, last_name))))
+            f.write('FN:{} {}\n'.format(first_name, last_name)
```
Comment (Lonami) [inline telethon/telegram_client.py]: Nice bug, missing the last parenthesis.

**Pull request merged**